Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN conda env update -n python2 -f .setup/environment-python2-cellprofiler.yml
 
 # Install prerequisites to install R
 RUN apt-get update && \
-    apt-get -y install libssl-dev libxml2-dev libcurl4-openssl-dev libpcre3 libpcre3-dev liblzma-dev libbz2-dev libjpeg-dev libssh2-1-dev
+    apt-get -y install libssl-dev libxml2-dev libcurl4-openssl-dev libpcre3 libpcre3-dev liblzma-dev libbz2-dev libjpeg-dev libssh2-1-dev libtiff-dev libpng-dev libfftw3-dev
```

Comment:
Optional: could you split this so there's one package per line, so that future diffs are easier to read?
```
RUN apt-get update && \
    apt-get -y install \
        libssl-dev \
        libxml2-dev \
        ...
```
Similarly for the `RUN Rscript -e` statement below